web/modules/weather_blocks/src/Plugin/Block/Test/DailyForecastBlock.php.test

```diff
@@ -24,7 +24,7 @@ final class DailyForecastBlockTest extends Base
      * Test that the block returns the expected data if we're on a location route.
      * @group unit
      * @group block
-     * @group daily-forecast-block
+     * @group wx-daily-forecast-block
```

Comment:
```suggestion
     * @group daily-forecast-block
```
These test groups refer to the Drupal block names, not CSS classes. They're only used to target tests to particular components instead of having to run the whole test suite.